src/mageia.ipxe

```diff
@@ -11,6 +11,7 @@ goto ${menu} ||
 :mageia
 set os Mageia
 menu Mageia - ${arch} - Image Sig Checks: [${img_sigs_enabled}]
+item 7 ${os} 7
 item 6 ${os} 6
 item 5 ${os} 5
 item 4 ${os} 4
```

Comment:
We should remove 4 and 5 because they are no longer supported.

Reply:
6 is also not supported.`It will be supported with security and bug fix updates for 18 months, up to 16 January 2019.` Source: https://wiki.mageia.org/en/Mageia_6_Release_Notes

Reply:
Also, is it now on 7.1?

http://mirror.dacentec.com/mageia/distrib/7.1/x86_64/install/images/Mageia-7.1-netinstall-x86_64.iso

Reply:
I have added 7.1 and removed all other releases.